### DISCUSSION TITLE
Periodic 3D mesh from a periodic labeled image

### DISCUSSION THREAD

**Discussion by oarcelus** [CGAL/cgal#7851] [Q&A]
Dear Forum,

From the presented examples it is clear the CGAL excels at obtaining smooth well behaved meshes for image stacks (labeled and unlabeled). But I was wondering if at the present time it is able to build a periodic 3D mesh from a periodic labeled image, where voxels of different domains are periodic in x y and z (or any combination).

**Comment by MaelRL** [comment]
Right now in master, there is nothing for periodic images. It might be done with a couple of wrappers. I will try to do it (when I have a bit of time).

There is no way to do periodicity in only a subset of directions.
